src/assets/scripts/client/navigationLibrary/StandardRoute/StandardRouteModel.js

```diff
@@ -291,6 +291,25 @@ export default class StandardRouteModel extends BaseModel {
     }
 
     /**
+     * Returns an array of unique fix names used by any portion of the procedure
+     *
+     * @for StandardRouteModel
+     * @method getAllFixNamesInUse
+     * @return {array<string>} ['fixname', 'fixname', 'fixname', ...]
+     */
+    getAllFixNamesInUse() {
+        const allFixNames = [
```

Comment:
what happens if any one of these collections is `undefined`?

I think this would only be a problem during development of a new airport, but this may be an edge case worth writing a test for?